addons/mail/models/mail_thread.py

```diff
@@ -2821,11 +2821,10 @@ def _notify_by_email_get_base_mail_values(self, message, additional_values=None)
                 ('model', '=', message_sudo.model), ('res_id', '=', message_sudo.res_id),
                 ('message_type', 'in', outgoing_types),
                 ('id', '!=', message_sudo.id),
-                ('subtype_id', '!=', note_type.id),  # filters out notes, using subtype which is indexed
```

Comment:
Keep at least messages with subtype_id being set. Pure logs have False as subtype, that is going to filter them out.